socketio/src/socket.rs

```diff
@@ -158,6 +158,47 @@ impl Socket {
         attachments.push(bin_data);
     }
 
+    pub(crate) fn decode_binary_payload(
```

Comment:
nit: why did this move? It's only used in raw_client

Reply:
I found encode is in socket.rs, so I move decode in socket.rs too.